packages/swap/prisma/migrations/20250110133349_add_max_boost_fee_to_swap_request/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "public"."SwapRequest" ADD COLUMN     "maxBoostFeeBps" INTEGER;
```

Comment:
we can backfill this column value by reading it from the related channel.

Reply:
thats a good idea - will do!